packages/@adobe/spectrum-css-temp/components/search/index.css

```diff
@@ -67,11 +71,15 @@ governing permissions and limitations under the License.
     -webkit-appearance: none;
   }
 
-  .spectrum-Textfield.spectrum-Search.is-invalid & {
+  .spectrum-Textfield.spectrum-Search--invalid & {
```

Comment:
ah, I see where this (is-invalid) came from, I think spectrum-css used it as a utility class
since we have css modules though, i think the change I suggested makes more sense